Instastories.py

```diff
@@ -56,7 +56,6 @@ def download_today_stories(arr_ids, cookie, number_of_persons):
         arr_ids (List): List of ids of people we want to get the stories.
         cookie (dict): Instagram Cookie for authentication in the requests.
     """
-    FOLDER = "ig_media"
```

Comment:
Leave folder here, get the value from settings['path'] or "ig_media" if the file is missing